src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeStorageClient.java

```diff
@@ -189,9 +185,9 @@ default void uploadWithPresignedUrlWithoutConnection(
   {
     if (!requirePresignedUrl())
     {
-      throw new SnowflakeSQLException(
+      throw new SnowflakeSQLLoggedException(
           SqlState.INTERNAL_ERROR,
-          ErrorCode.INTERNAL_ERROR.getMessageCode(),
+          ErrorCode.INTERNAL_ERROR.getMessageCode(), null,
```

Comment:
You can pass session to this function.

Reply:
There is no session for this particular function **uploadWithPresignedUrlWithoutConnection()** because there is no connection made, and therefore no valid session.